x-pack/platform/plugins/shared/streams_app/public/components/data_management/preview_table/index.tsx

```diff
@@ -162,6 +164,9 @@ export function PreviewTable({
       aria-label={i18n.translate('xpack.streams.resultPanel.euiDataGrid.previewLabel', {
         defaultMessage: 'Preview',
       })}
+      data-test-subj={
```

Comment:
I think this is not necessary as the failing test relates to the left panel (processors list). there's already a function that waits for that list to show up, imo we should increase timeout there instead https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/streams_app/test/scout/ui/fixtures/page_objects/streams_app.ts#L298

Reply:
And if there are failures with the preview we should increase timeout here https://github.com/elastic/kibana/blob/main/x-pack/platform/plugins/shared/streams_app/test/scout/ui/fixtures/page_objects/streams_app.ts#L333